server/auth-router.js

```diff
@@ -101,9 +101,12 @@ module.exports = function authRouter(config = {}) {
 		info(`LoginUI: execute logout, session id:${req.sessionID}, cookie:${getSyncCookie(req)}, user id:${req.user && req.user.id}`); // eslint-disable-line max-len
 		const returnUrl = encodeURIComponent(`https://${config.host}`);
 		const logoutUrl = `https://${config.auth0.domain}/v2/logout?returnTo=${returnUrl}`;
-		req.logout(); // removes req.user
-		noteLoggedOut(res);
-		res.redirect(logoutUrl);
+		req.logout({}, err => {
+			console.error(err);
+			// removes req.user
+			noteLoggedOut(res);
+			res.redirect(logoutUrl);
```

Comment:
So this redirect is only needed if there's an error?

Reply:
`err` is the only param passed into this callback.

Reply:
So it provides an error if there is one, but the same callback is used for success?

Reply:
Yea, providing a callback is now required. Without it there were errors

Reply:
In that case, do we want an `if (err)` around the `console.error`?

Reply:
Yes I think that's a good callout.